.storybook/preview.js

```diff
@@ -31,7 +31,7 @@ export const parameters = {
 };
 
 const GlobalStyle = createGlobalStyle`
-  :focus {
+  :focus-visible {
```

Comment:
This is necessary to prevent programmatic focus from showing an indicator (e.g., menu focuses items even when using a mouse).